### ISSUE TITLE
Triggered abbreviation is removed but phrase is blank.

### ISSUE THREAD

**Issue by apcrcom** [autokey/autokey#343]
## Classification:

UI/Usability

## Reproducibility:

Always

## Version

AutoKey version: 0.90.4

Used GUI (Gtk, Qt, or both): 

XFCE

Installed via: synaptic

Linux Distribution:

MX

Summary of the problem.

## Steps to Reproduce (if applicable)

- FILE -> NEW -. HRASE -> "x" -> OK
- (enter phrase contents) LALALALA
- Abbreviations -> SET -> ADD -> .svp (enter) 
----------Trigger on : All Non-Word
----------Remove Typed Abbreviation
- OK

## Expected Results

- Go to text field, type .svp = LALALALA

## Actual Results

- Go to text field, type .svp = 

(typed abbreviation removes but there is blank not text in its place)

If helpful, submit screenshots of the issue to help debug.\

https://i.imgur.com/XTOWPly.png

Debugging output, obtained by launching autokey via `autokey-gtk --verbose` (or `autokey-qt --verbose`, if you use the Qt interface) is also useful.\
Please upload the log somewhere accessible or put the output into a code block (enclose in triple backticks).
```
/usr/lib/python2.7/dist-packages/autokey/gtkapp.py:24: PyGIWarning: Gtk was imported without specifying a version first. Use gi.require_version('Gtk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk, GObject, GLib
/usr/lib/python2.7/dist-packages/autokey/gtkui/notifier.py:19: PyGIWarning: Notify was imported without specifying a version first. Use gi.require_version('Notify', '0.7') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Gdk, Notify
/usr/lib/python2.7/dist-packages/autokey/gtkui/notifier.py:28: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3
/usr/lib/python2.7/dist-packages/autokey/gtkui/configwindow.py:20: PyGIWarning: GtkSource was imported without specifying a version first. Use gi.require_version('GtkSource', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import Gtk, Pango, GtkSource, Gdk, Gio
2019-12-22 10:30:50,966 DEBUG - root - AutoKey is already running as pid 11847

```

**Comment by BlueDrink9** [comment]
I couldn't reproduce this on the latest master commit

**Comment by BlueDrink9** [comment]
Could you try running the latest master, and try it from the other GUI?

Also, double check the phrase contents are not actually blank.

**Comment by BlueDrink9** [comment]
Actually yeah, we're up to 0.95.9 currently, so 0.90 is ancient. If you're having bugs, upgrade. If it keeps occuring, then this issue is worth looking at.

See [Joseph's answer](https://github.com/autokey/autokey/issues/324#issuecomment-554583876) for more info about versions.

**Comment by apcrcom** [comment]
Ungh.... Im using MX, and when I went to install the .deb from the website, gdebi came up with a medssage saying "better to use the one from the repo". So I used synaptic and thats what it gave me. 

Using the latest version worked... of course. Thanks for looking at this, and BTW, the applicattion is perfect for what I need it for. Thanks for developing it and making it available to for plebs like me to download.
